libsmtutil/Z3CHCInterface.cpp

```diff
@@ -97,7 +102,8 @@ pair<CheckResult, CHCSolverInterface::CexGraph> Z3CHCInterface::query(Expression
 	}
 	catch (z3::exception const& _err)
 	{
-		if (_err.msg() == string("max. resource limit exceeded"))
+		set<string> msgs{"max. resource limit exceeded", "canceled"};
```

Comment:
This tricked me again: https://www.merriam-webster.com/words-at-play/canceled-or-cancelled

Reply:
Just copied directly from the solver output